cmd/asset-syncer/utils.go

```diff
@@ -215,10 +216,18 @@ func (r *HelmRepo) FetchFiles(name string, cv models.ChartVersion) (map[string]s
 	}, nil
 }
 
+// TagList represents a list of tags as specified at
+// https://github.com/opencontainers/distribution-spec/blob/master/spec.md#content-discovery
+type TagList struct {
+	Name string   `json:"name"`
```

Comment:
yes, that's correct, the `Name` within the struct contain the registry name (or directory) as well. Note that from the user point of view, it would be the same to specify `registry: docker.io repositories: bitnami/apache, bitnami/jenkins` than `registry: docker.io/bitnami repositories: apache, jenkins`.

Reply:
I don't know if it has an impact here (don't think so), but we should remember that other repos are including slashes as part of the name (e.g., from `registry: repo.chartcenter.io repositories: bitnami/wordpress`)